smithy-swift-codegen/src/main/kotlin/software/amazon/smithy/swift/codegen/integration/HttpProtocolTestGenerator.kt

```diff
@@ -88,7 +89,9 @@ class HttpProtocolTestGenerator(
             val testFilename = "./${ctx.settings.moduleName}Tests/$testClassName.swift"
             ctx.delegator.useTestFileWriter(testFilename, ctx.settings.moduleName) { writer ->
                 LOGGER.fine("Generating request protocol test cases for ${operation.id}")
-
+                for (import in imports) {
+                    writer.addImport(import)
+                }
```

Comment:
the way i'm reading this, `imports` is a constant empty list -- what is this doing?

Reply:
cuz we enable service protocol tests like Glacier that takes in service specific middlewares that live in `AWSClientRuntime`